### ISSUE TITLE
List of branches allow to filter but the filter is not applied

### ISSUE THREAD

**Issue by tesonep** [npasserini/iceberg#1455]
![image](https://user-images.githubusercontent.com/355455/110948805-16e33000-8342-11eb-8f38-dd49eb1cda1e.png)

**Comment by tinchodias** [comment]
Ah, we can implement it as in `IceTipWorkingCopyBrowser`. IIRC, @estebanlm told me that table filtering should be converted like that in Spec2 (i.e. filtering in tables won't be a behavior provided by the Spec2 table, but instead, you add a input text field and configure the search externally)

**Comment by tesonep** [comment]
We can do our own presenter for that, and if it is nice, we can sell it to the Spec2 project.

**Comment by estebanlm** [comment]
nah, why?... you now have `SpFilteringListPresenter` as an example on how to do it (most of the cases like the one of the screenshot, you just need to use it, others you can extend it).
